### DISCUSSION TITLE
analysing-my-youtube-subscriptions/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#6] [Comments]
# Analysing my YouTube subscriptions & views over the last 4 years (and how you can too!) | Jake Lee

According to Wallace Boyer in Chuck Palahniuk’s “Rant”: “… every person is obsessed with himself. You are your own favorite hobby. You’re an expert on you”. With that in mind, I spent some time exploring my YouTube viewing / subscribing habits over the last few years!

http://localhost:4000/analysing-my-youtube-subscriptions/

**Comment by JakeSteam** [comment]
test via giscus
